jbmc/src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -753,6 +753,8 @@ void java_bytecode_convert_classt::convert(
     else
       component.set_access(ID_default);
 
+    component.set_is_final(f.is_final);
```

Comment:
:bulb: If the class is opaque the `is_final` flag should be set to true.

Reply:
After discussion with @thomasspriggs, we worked out that dealing with fields of opaque classes may not be as straightforward than I thought. So my comment can be ignored.